modules/core/src/main/scala/higherkindness/droste/algebras.scala

```diff
@@ -17,6 +17,11 @@ import cats.syntax.flatMap._
 import cats.syntax.functor._
 import cats.syntax.semigroupal._
 
+/** A `GAlgebra[F[_], S, A]` is a wrapper for a function from `F[S]` to `A`.
+  *
+  * This type is isomorphic to a `cats.data.Cokleisli[F, S, A]`, but
+  * unlike Cats we use an AnyVal for efficiency.
```

Comment:
I'd remove the note about efficiency unless we show that it's actually more efficient in a meaningful way.

Reply:
The efficiency was more of a lucky guess. I was wondering, given that `droste` is built on `cats`, why would you define yet another name for the same thing; as opposed to a simple type alias `type GAlgebra[F[_], S, A]  = cats.data.Cokleisli[F, S, A]`